src/cloud/social/alias/randombytes.js

```diff
@@ -25,5 +25,9 @@ module.exports = function(size) {
       buffer[i] = Math.floor(Math.random() * 256);
     }
   }
-  return buffer;
+  if (cb) {
```

Comment:
Having a callback could simplify switching to the cryptographically secure core.crypto fill. I won't assert whether or not that's something that needs to be done in *this* pull request, but I do think it's something worth getting done soon, as Math.random is pretty bad.